docs/user/installation.rst

```diff
@@ -121,6 +133,53 @@ PyPi
 
 `intelmqsetup` will create all necessary directories, provides a default configuration for new setups. See the :ref:`configuration` for more information on them and how to influence them.
 
+Docker **with** docker-compose (recommended)
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Navigate to your preferred installation directory, i. e. use ``mkdir ~/intelmq && cd ~/intelmq``
+
+.. code-block:: bash
+
+   git clone https://github.com/certat/intelmq-docker.git
+
+   sudo docker pull certat/intelmq-full:1.0
+
+   mkdir intelmq_logs
```

Comment:
If we do `mkdir intelmq_logs` in the parent directory of the repository, but then run `docker-compose up` *in* the repository, the container won't be able to access the `intelmq_logs` folder, because the `docker-compose.yml` file points to `./intelmq-logs` which means the folder should be in the directory the `docker-compose.yml` file resides in. It seems docker-compose creates that directory with root permissions if it does not exist, therefore I would simply ship an empty `intelmq_logs` directory in the git repository.

When testing this I also had to change the image path in the `docker-compose.yml` file from `intelmq-full:1.0` to `certat/intelmq-full:1.0`.

Reply:
1) Done.
2) Well, if you want to build your own image you should not use `certat/` namespace. Image should be built without any namespaces, i. e. `intelmq-full:1.0`

Reply:
Changed `intelmq-full:1.0` to `certat/intelmq-full:1.0` as default. Added additional developer note in README